pkg/kapp/crdupgradesafety/change_validator_test.go

```diff
@@ -0,0 +1,429 @@
+// Copyright 2024 The Carvel Authors.
+// SPDX-License-Identifier: Apache-2.0
+package crdupgradesafety
```

Comment:
nit: newline after copyright headers

Reply:
Also, test package name should be `crdupgradesafety_test`. I think I missed this in previous PRs (apologies 😅).

Reply:
Should be done in https://github.com/carvel-dev/kapp/pull/921/commits/ce52021254d7ca39becbd0dd4711c11680b1db55

Had to make some slight modifications to some types in the `crdupgradesafety` package to export fields needed for testing